src/main/resources/static/api.yaml

```diff
@@ -88,4 +141,88 @@ components:
       properties:
         hipId:
           type: string
-          description: Identifier of the health information provider where patient record needs to be looked up.
\ No newline at end of file
+          description: Identifier of the health information provider where patient record needs to be looked up.
+
+    ConsentRequestsRepresentation:
+      type: object
+      properties:
+        limit:
+          type: integer
+        size:
+          type: integer
+        offset:
+          type: integer
+        requests:
+          type: array
+          maxItems: 100
+          items:
+            $ref: '#/components/schemas/ConsentRequestRepresentation'
+
+    ConsentRequestRepresentation:
+      type: object
+      properties:
+        purpose:
+          type: object
+          properties:
+            description:
+              type: string
+        hitypes:
+          type: array
+          items:
+            $ref: '#/components/schemas/HITypeRepresentation'
+        requester:
+          type: object
+          properties:
+            id:
+              type: string
+              format: uuid
+              example: 496413f0-7b63-4f80-9bb0-9382cc3108d3
+            name:
+              type: string
+              example: "Dr. Karuppan"
+            orgName:
```

Comment:
I would pull out organization from inside requester, and have a top level HIU entry. Check also https://app.swaggerhub.com/apis/NCG5/hdaf_api/0.9#/ConsentsRequest

- also we can keep identifier as object, and similar to other identifier of entities like patient, practitioner. Identifier having value, type, and system. 
- if we are inclined to keep single value of "identifier", then we must be explicit in our documentation that this identifier's value is same as that in the Central Registry.